src/Arrays/matrix_vector_operations.jl

```diff
@@ -20,3 +20,57 @@ function inner(x::AbstractVector{N}, A::AbstractMatrix{N}, y::AbstractVector{N}
               ) where {N}
     dot(x, A * y)
 end
+
+"""
+    _vector_type(T)
+
+Return a corresponding vector type with respect to type `T`.
+
+### Input
+
+- `T` -- vector type
```

Comment:
```suggestion
- `T` -- matrix type
```

Reply:
Actually, this should be either vector or matrix type.